__tests__/app.test.js

```diff
@@ -331,4 +331,27 @@ describe('DELETE /api/comments/:comment_id', () => {
       })
     })
   })
-  
\ No newline at end of file
+  describe('GET /api/users', () => {
+    test('status 200, will return an array of objects containing the correct properties ', ()=> {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body }) => {
+            const { users } = body
+            expect(users).toHaveLength(4)
+            users.forEach((user) => {
+                expect(user).toHaveProperty('username', expect.any(String));
+                expect(user).toHaveProperty('name', expect.any(String));
+                expect(user).toHaveProperty('avatar_url', expect.any(String));
+              });
+            })
+    })
+    test('status 404, responds with path not found when passed incorrect path',() =>{
+        return request(app)
+        .get('/api/nousers')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Not found");
+          });
+    })
```

Comment:
This test isn't specific to this endpoint - if you already have any test that checks a bad path you can remove it altogether. Whether the path is /api/nousers or /banana - it's the same behaviour.